modyn/selector/internal/selector_strategies/abstract_selection_strategy.py

```diff
@@ -390,3 +398,6 @@ def _persist_samples(self, keys: list[int], timestamps: list[int], labels: list[
 
         for proc in processes:
             proc.join()
+
+    def get_uses_weights(self) -> int:
+        return self._uses_weights
```

Comment:
I suppose if this is a public member we don't need this getter. Instead, just rename `_uses_weights` to `uses_weights`, which is more Pythonic (private members start with _, public members dont)

Reply:
(I see that it is on every strategy, but then every strategy just needs to have this member, initialized by the AbstractSelectionStrategy)

Reply:
Yep, changed. I added the method to avoid the confusion method/variable that was in the previous post.